src/style.css

```diff
@@ -1,3 +1,71 @@
 body {
   margin: 0;
 }
+
+/* navigation */
+
+header {
+  --header-height: 60px;
```

Comment:
[[checklist](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md#css)] Don't fix the container size. Links should determine its height